### PR TITLE
Update the way the words are replaced and the way the replacer works

### PR DESCRIPTION
**Pull request by hmgibson23**
Also add a couple more words that we might want to black list

### CODE REVIEW

File: src/scribe-plugin-ngrams.js [hunk 1]
```diff
@@ -5,34 +5,24 @@ module.exports = function(config) {
...
 
         nGramsCommand.execute = () => {
-            const text = unWords.reduce((val, fn) => {
-                return fn(val);
+            const text = unWantedWords.reduce((current, replace) => {
+                return replacer(replace, current);
             }, scribe.el.innerText);
 
             const bigrams = NGrams.bigrams(text);
+            debugger;
```
Comment (theefer) [inline src/scribe-plugin-ngrams.js]: Meh

File: src/scribe-plugin-ngrams.js [hunk 2]
```diff
@@ -5,34 +5,24 @@ module.exports = function(config) {
...
-            (text) => text.replace(/ had /g, " "),
-            (text) => text.replace(/ gone /g, " "),
-            (text) => text.replace(/ that /g, " "),
-            (text) => text.replace(/ from /g, " "),
-            (text) => text.replace(/ to /g, " ")
-        ];
+        const unWantedWords = [" the ", " and ", " but ", " then ", " a ", " of ",
+                              " than ", " an ", " in ", " on ", " by ", " he ", " she ",
+                              " had ", " gone ", " that ", " from ", " that ", " from ",
+                              " to ", " who ", " whose "];
```
Comment (theefer) [inline src/scribe-plugin-ngrams.js]: If always surrounded by spaces, could have just the words and adding the spaces programmatically?

**Comment by theefer** [comment]
:+1: sensible

File: src/scribe-plugin-ngrams.js (hunk 2 above)
Comment (hmgibson23) [inline src/scribe-plugin-ngrams.js]: Good idea.

**Pull request merged**